<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GetInfo</class>
 <widget class="QWidget" name="GetInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>状态：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonMan">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>411</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>年龄段</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>弹窗显示</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Age0_14">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0-14</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Age14_18">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>14-18</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Age18_24">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>18-24</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Age24_35">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>24-35</string>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButtonYellow">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>黄码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>性别：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonGreen">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>绿码</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonWoman">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonRed">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>红码</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
